internal/organizations/retrieve.go

```diff
@@ -44,12 +44,11 @@ func (c *Client) Retrieve() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			include := []string{"Inner_example"}
-			exclude := []string{"Inner_example"}
-			withoutProjects := "withoutProjects_example"
```

Comment:
The `withoutProjects` option did not exist prior to `metal-go` conversion, and is not wired up to any user-facing flags, so I've opted to remove it rather than fix the type error for now.